variants/m5stack_atom/pins_arduino.h

```diff
@@ -3,15 +3,21 @@
 
 #include <stdint.h>
 
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       16
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(esp32_adc2gpio[(p)]):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
```

Comment:
Please remove this part from all pins_arduino.h files of your boards.
These #defines are now defined globally per SoC.

Reply:
Thanks for your reply, new changes have been committed now